libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -1383,6 +1383,7 @@ void GCS_MAVLINK::send_autopilot_version() const
     uint16_t vendor_id = 0;
     uint16_t product_id = 0;
     uint64_t uid = 0;
+    uint8_t  uid2[18] = {0};
```

Comment:
this should be:
`uint8_t uid2[MAVLINK_MSG_AUTOPILOT_VERSION_FIELD_UID2_LEN] = {0};`

Reply:
Many thanks for that Tom. I fixed some other fields as well now.